libs/sync/sync.go

```diff
@@ -47,8 +52,9 @@ type Sync struct {
 	filer    filer.Filer
 
 	// Synchronization progress events are sent to this event notifier.
-	notifier EventNotifier
-	seq      int
+	notifier        EventNotifier
+	outputWaitGroup *stdsync.WaitGroup
+	seq             int
```

Comment:
Please move the outputWaitGroup below with its own comment.

The sync type didn't spawn its own routine before, and it was possible to call `Events()`, then `RunOnce()`, then `Close()`, and repeat. This is no longer possible now that we construct the events channel from the constructor. I suspect calling `Events()` will actually be problematic now.

Reply:
Removed the `Events` completely, as it's not used now